src/components/App.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import "../css/App.css";
+import Scoreboard from "./Scoreboard";
+import Field from "./Field";
+import Homebase from "./Homebase";
+import Counter from "./Counter";
+
+function App() {
+  return (
+    <div className="App">
+      <header>
+        <Scoreboard />
+      </header>
+      <main>
+        <Field className="north-field" />
+        <Field className="west-field" />
+        <Field className="east-field" />
+        <Field className="south-field" />
+        <Homebase />
+      </main>
+      <footer>
+        <Counter />
+      </footer>
```

Comment:
I'm not sure semantically what elements best fit these components. I know the counter goes on the bottom, but I don't know if that means it's really a footer.